content/kubermatic/main/architecture/concept/kkp-concepts/networking/_index.en.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Networking"
+date = 2022-04-20T12:15:02+02:00
+weight = 20
+
++++
+
+## KKP Networking 
+
+In the Kubermatic Kubernetes Platform (KKP), some parts of the Kubernetes control plane are deployed in the dedicated Seed cluster instead of the user cluster. 
+This design requires two endpoints to be exposed at the Seed via the nodeport-proxy service in order to be accessible from user clusters. 
+These endpoints expose the Kubernetes apiserver as well as a proxy to connect to the user cluster network for each user cluster on a Seed. 
+The [expose strategy]({{< ref "../../../../tutorials-howtos/networking/expose-strategies" >}}) enables configuration of how the services are exposed on a Seed.
+
+This section explains how the connection between user clusters and the control plane is established, as well as the general networking concept in KKP.
+
+KKP Operator &rarr; Master cluster Kubernetes API
+KKP Operator &rarr; Seed cluster Kubernetes API
+
+Kubernetes API &rarr; Seed Cluster Kubernetes API
+Kubernetes API &rarr; Seed cluster nodeport-proxy 
+
+Seed controller manager &rarr; Seed cluster Kubernetes API
+Seed controller manager &rarr; Cloud Provider API
```

Comment:
I don't really know what this section is trying to tell me to be honest. What is it supposed to say/explain?